src/db/sql/sqlSelectBuilder.ts

```diff
@@ -51,6 +63,9 @@ export class SqlSelectBuilder extends SqlBuilder {
   build(): string {
     const parts: Array<string> = [`SELECT ${this._select.join(', ')}`, `FROM ${this._from.join(' ')}`]
 
+    if (!Objects.isEmpty(this._join) && !Objects.isEmpty(this._on)) {
+      parts.push(`JOIN ${this._join.join(' ')} ON ${this._on.join(' ')}`)
+    }
```

Comment:
if we have several joins, won't this produce something like (where o is the source table):
```sql
join schema.tableA as a schema.tableB as b ON
o.a = a.a o.b = b.b 
```
would't we need to have a JoinConditionBuilder?